core/templates/dev/head/domain/utilities/LanguageUtilService.js

```diff
@@ -78,6 +78,7 @@ oppia.factory('LanguageUtilService', [
       },
       supportsAutogeneratedAudio: function(explorationLanguageCode) {
         return (
+          speechSynthesis.getVoices().length !== 0 &&
```

Comment:
Maybe also add a speechSynthesis before this, to handle things like IE? I.e. `speechSynthesis && speechSynthesis.getVoices().length && ...`

Also there's something to be said for moving this to BrowserCheckerService since it's mainly browser dependent. How about BrowserCheckerService.supportsSpeechSynthesis()?

Reply:
@seanlip, Adding `'speechSynthesis' in window && speechSynthesis.getVoices().length &&....`  that way we'll not get refrenceerror in console for the browsers not supporting speechSynthesis... see [here](https://developers.google.com/web/updates/2014/01/Web-apps-that-talk-Introduction-to-the-Speech-Synthesis-API#demo)

Reply:
Yep, sounds good. Can you create a BrowserCheckerService.supportsSpeechSynthesis() method that is basically:

    window.hasOwnProperty('speechSynthesis') &&
    speechSynthesis.getVoicees().length && ...

? Also, if isChrome() isn't used any more then let's remove it.

Reply:
Safari also has speechSynthesis.getVoices().length = 49 but unfortunately it's not supporting right now i.e, not playing audio.. @nithusha21 is investigating it once he'll resolve the issue we can remove isChrome().
Also, we just use specchSynthesis to speak in 'en-GB' so 
```
$scope.supportsSpeechSynthesis = function(){ 
  supportLang = false;
  if(window.hasOwnProperty('speechSynthesis')){
    speechSynthesis.getVoicees().forEach(function(voices) {
    if (voice.lang == 'en-GB') {
      supportLang = true;
    };
    });
  }
  return supportLang;
}
```

Reply:
OK, seems reasonable! Thanks @DubeySandeep.